sample/DocumentDB-Node/index.js

```diff
@@ -0,0 +1,9 @@
+﻿module.exports = function (context, input) {
+    context.log('Node.js triggered function via EasyTables called with input ' + input);
```

Comment:
Update the description used in the log since the trigger is a queue and the output is DocumentDB